packages/data-point/lib/reducer-types/reducer-helpers/utils/index.test.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-env jest */
+
+const utils = require('./index')
+
+describe('utils#isFalsy', () => {
+  expect(utils.isFalsy(null)).toBe(true)
```

Comment:
These should be wrapped in a `test` block.

Reply:
ok - do you mean I should replace `describe`with `test`, or have both `describe` and `test`?

Reply:
The second one. There should be a `test` block nested inside the `describe` block. Really the `test` method is required but the `describe`. https://facebook.github.io/jest/docs/en/api.html#testname-fn-timeout

Also, just like the other comment each of these `expect` expressions should probably be a separate test.